lib/matplotlib/colors.py

```diff
@@ -3097,8 +3097,13 @@ def rgb_to_hsv(arr):
         dtype=np.promote_types(arr.dtype, np.float32),  # Don't work on ints.
         ndmin=2,  # In case input was 1D.
     )
+
     out = np.zeros_like(arr)
     arr_max = arr.max(-1)
+    # Check if input is in the expected range
+    if np.any(arr_max > 1) or np.any(arr < 0):
+        raise ValueError("Input RGB values must be in the range [0, 1]. "
+                         f"Found values out of range in array: {arr}")
```

Comment:
Displaying the whole array generally does not make sense. Either leave out the value completely or state the offending max or min value; for example "Input RGB values must be in the range [0, 1]. Found a maximum of 253" may be a useful hint. When going with the values, you want to do two separate if statements and issue tailored messages.

Reply:
I see. I will change the code to address the issue with the error messages